<launch>
    <arg name="drone_id"  value="0"/>

    <!-- trajectory server -->
    <!-- Outputs mavros command to px4 -->
    <node pkg="ego_planner" name="drone_$(arg drone_id)_traj_server" type="traj_server_node" output="screen">
        <remap from="~planning/trajectory" to="drone_$(arg drone_id)_planning/trajectory"/>
        <param name="traj_server/time_forward" value="1.0" type="double"/>
        <!-- Frequency to publish command -->
        <param name="traj_server/pub_cmd_freq" value="25.0" type="double"/>
        <param name="traj_server/state_machine_tick_freq" value="50.0" type="double"/>
        <param name="traj_server/takeoff_height" value="1.0" type="double"/>
    </node>


</launch>
